test/verify/packagelib.py

```diff
@@ -156,13 +158,15 @@ def createDeb(self, name, version, depends, postinst, install, content, arch, pr
                     self.machine.execute(f"cp '{data['path']}' '{dest}'")
                 else:
                     self.machine.write(dest, data)
-        cmd = '''mkdir -p /tmp/b/DEBIAN {repo}
-                 printf "Package: {name}\nVersion: {ver}\nPriority: optional\nSection: test\nMaintainer: foo\nDepends: {deps}\nArchitecture: {arch}\nDescription: dummy {name}\n{provides}\n" > /tmp/b/DEBIAN/control
-                 {post}
-                 touch /tmp/b/stamp-{name}-{ver}
-                 dpkg -b /tmp/b {deb}
-                 rm -r /tmp/b
-                 '''.format(name=name, ver=version, deps=depends, deb=deb, post=postinstcode, repo=self.repo_dir, arch=arch, provides=provides)
+        cmd = f'''mkdir -p /tmp/b/DEBIAN {self.repo_dir}
+                  printf "Package: {name}\nVersion: {version}\nPriority: optional\nSection: test
+Maintainer: foo\nDepends: {depends}\nArchitecture: {arch}
```

Comment:
I had earlier rewritten this in a very nice way.  Something like:

```
    printf "%s: %s\n" \
        Maintainer  'foo'
        Depends '...' \
        .... \
    > /tmp/...
```

which allows keeping everything neatly intended.

Reply:
I don't like this that much, as it requires a long list of `%s: %s` templates, and keep their number in sync with the fields. I used a slightly different form now with dedent() and m.write().

Reply:
```
$ printf "%s: %s\n" a b c d e f g h
a: b
c: d
e: f
g: h
```

Reply:
TIL! However, I honestly don't feel like cycling this again, unless you insist. IMHO dedent() is quite decent as well.

Reply:
Sure.  I agree that it looks fine now with dedent.  Maybe even better.